### DISCUSSION TITLE
Can JSBSim simulate  Immelmann turn with flightgear？

### DISCUSSION THREAD

**Discussion by theorynice** [JSBSim-Team/jsbsim#417] [Q&A]
As mentioned above, I have tried to fly an Immelmann turn or kulbit fly by F15 in FlightGear but failed. Has anybody succeeded?

**Comment by bcoconni** [comment]
I think you'd rather open an issue in the GitHub repo for FlightGear's F15 and f-14b aircraft: https://github.com/Zaretto/fg-aircraft. @Zaretto - who is the author of the F15 aircraft (and its JSBSim FDM) in FlightGear - should most likely be able to answer your question. Of course, if @Zaretto and yourself narrow down the issue to being a problem in JSBSim you are most welcome to come back here and open an issue.

Thanks.

**Reply by theorynice** [reply]
Thank you very much. It helps me so much.
